Dockerfile

```diff
@@ -0,0 +1,13 @@
+# docker build -t rc:latest .
+# docker run -v `pwd`:/rc rc:latest
+
+FROM node:lts-jessie
+
+RUN npm install --global web-ext
+
+RUN mkdir /rc
+
+WORKDIR /rc
+
+ENTRYPOINT ["./build/extension-chrome/build.sh"]
+ENTRYPOINT ["./build/extension-firefox/build.sh"]
```

Comment:
This is overriding the `ENTRYPOINT` rather than running it twice. You should add some kind of 'build-all` script that calls both these build scripts if you want to run them together

Reply:
Add `build_all.sh` and updated Dockerfile to point to that as the ENTRYPOINT.